src/main/java/org/isf/generaldata/ExaminationParameters.java

```diff
@@ -95,6 +95,9 @@ public final class ExaminationParameters extends ConfigurationProperties {
 	
 	public static int SAT_MIN;
 	private static final int DEFAULT_SAT_MIN = 50;
+	
+	public static int SAT_MAX;
+	private static final int DEFAULT_SAT_MAX = 100;
```

Comment:
This is not needed because saturation cannot be above 100%

Reply:
I use that value to check if saturation give by use is less than that Max value

Reply:
I understand, but it is not something that can be decided by the user, it is a percentage :)
so I think you can just hardcode it or at least don't allow the user to modify it like other parameters in line 156

Reply:
okay can I fix value 99 to that param

Reply:
why 99? saturation can be 100 and it could means hyperventilation...

Reply:
you can just put hardcode 100 or if you want you can keep it and remove the line 156 further down